src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,17 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+interface Props {
+  user: User | null;
```

Comment:
```suggestion
  user: User;
```
You don't need to render `UserInfo` component if you don't have a user